GliaWidgets/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -69,7 +70,7 @@ class ChatViewModel: EngagementViewModel, ViewModel {
     private var unreadMessages: UnreadMessagesHandler!
     private let showsCallBubble: Bool
     private let storage = ChatStorage()
-    private let uploader = FileUploader()
+    private let uploader = FileUploader(maximumUploads: 25)
```

Comment:
I guess the requirements don't mention that this needs to be changed from the integrator side if possible, correct? If there's no mention of that, then this is fine.

Reply:
Correct.